### ISSUE TITLE
Plugin install failed

### ISSUE THREAD

**Issue by prikesh-patel** [fabmation-gmbh/helm-whatup#40]
I am trying to install the helm-whatup plugin on an M1 Macbook. When running the install command I get the following error

```shell
No prebuild binary for darwin-arm64.
Failed to install helm-whatup
-e 	For support, go to https://github.com/fabmation-gmbh/helm-whatup.
Error: plugin install hook for "whatup" exited with error
```

Is this plugin compatible with M1 Macbooks?

**Comment by l0nax** [comment]
Hey, we didn't compile and publish the plugin for `darwin/arm64`.
I added the binary file for version `v0.6.1`.

Could you please retry to install the plugin?

**Comment by prikesh-patel** [comment]
I'm still getting the error

```shell
No prebuild binary for darwin-arm64.
Failed to install helm-whatup
-e 	For support, go to https://github.com/fabmation-gmbh/helm-whatup.
Error: plugin install hook for "whatup" exited with error
```

I'm running the `helm plugin install https://github.com/fabmation-gmbh/helm-whatup` command.

**Comment by l0nax** [comment]
Oh, I made a mistake – I'm sorry. Please try it again.

**Comment by prikesh-patel** [comment]
Still getting the same error!

```shell
~ helm plugin install https://github.com/fabmation-gmbh/helm-whatup
No prebuild binary for darwin-arm64.
Failed to install helm-whatup
-e 	For support, go to https://github.com/fabmation-gmbh/helm-whatup.
Error: plugin install hook for "whatup" exited with error
```

**Comment by l0nax** [comment]
I'm sorry for the inconvenience.

Fixed with release [v0.6.3](https://github.com/fabmation-gmbh/helm-whatup/releases/tag/v0.6.3).

**Comment by prikesh-patel** [comment]
That's alright! But I'm getting a slightly different error now

```shell
~ helm plugin install https://github.com/fabmation-gmbh/helm-whatup
Downloading https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-amd64.tar.gz
https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-arm64.tar.gz
curl: (3) URL using bad/illegal format or missing URL
Failed to install helm-whatup
-e 	For support, go to https://github.com/fabmation-gmbh/helm-whatup.
Error: plugin install hook for "whatup" exited with error
```

**Comment by l0nax** [comment]
Could you please run the command with the `--debug` flag set and post the output?

`helm --debug plugin install https://github.com/fabmation-gmbh/helm-whatup`

**Comment by prikesh-patel** [comment]
The error has a different output for an unknown reason, but here is the `--debug` output.

```shell
~ helm --debug plugin install https://github.com/fabmation-gmbh/helm-whatup
[debug] vcs_installer.go:162: updating https://github.com/fabmation-gmbh/helm-whatup
[debug] vcs_installer.go:91: copying /Users/patelp1/Library/Caches/helm/plugins/https-github.com-fabmation-gmbh-helm-whatup to /Users/patelp1/Library/helm/plugins/helm-whatup
plugin_install.go:82: [debug] loading plugin from /Users/patelp1/Library/helm/plugins/helm-whatup
plugin.go:60: [debug] running install hook: /bin/sh -c $HELM_PLUGIN_DIR/install-binary.sh
+ PROJECT_NAME=helm-whatup
+ PROJECT_GH=fabmation-gmbh/helm-whatup
+ '[' -z '' ']'
+ export HELM_HOME=/Users/patelp1/.helm
+ HELM_HOME=/Users/patelp1/.helm
+ '[' '' == 1 ']'
+ trap fail_trap EXIT
+ set -e
+ initArch
++ uname -m
+ ARCH=arm64
+ case $ARCH in
+ initOS
++ tr '[:upper:]' '[:lower:]'
+++ uname
++ echo Darwin
+ OS=darwin
+ case "$OS" in
+ verifySupported
+ local 'supported=linux-amd64\ndarwin-amd64\ndarwin-arm64'
+ echo 'linux-amd64\ndarwin-amd64\ndarwin-arm64'
+ grep -q darwin-arm64
+ type curl
+ getDownloadURL
+ local latest_url=https://api.github.com/repos/fabmation-gmbh/helm-whatup/releases/latest
+ type curl
++ curl -s https://api.github.com/repos/fabmation-gmbh/helm-whatup/releases/latest
++ grep darwin
++ awk '/\"browser_download_url\":/{gsub( /[,\"]/,"", $2); print $2}'
+ DOWNLOAD_URL='https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-amd64.tar.gz
https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-arm64.tar.gz'
+ downloadFile
+ PLUGIN_TMP_FILE=/tmp/helm-whatup.tgz
+ echo 'Downloading https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-amd64.tar.gz
https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-arm64.tar.gz'
Downloading https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-amd64.tar.gz
https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-arm64.tar.gz
+ type curl
+ curl -L 'https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-amd64.tar.gz
https://github.com/fabmation-gmbh/helm-whatup/releases/download/v0.6.3/helm-whatup-0.6.3-darwin-arm64.tar.gz' -o /tmp/helm-whatup.tgz
curl: (3) URL using bad/illegal format or missing URL
+ fail_trap
+ result=3
+ '[' 3 '!=' 0 ']'
+ echo 'Failed to install helm-whatup'
Failed to install helm-whatup
+ echo -e '\tFor support, go to https://github.com/fabmation-gmbh/helm-whatup.'
-e 	For support, go to https://github.com/fabmation-gmbh/helm-whatup.
+ exit 3
Error: plugin install hook for "whatup" exited with error
helm.go:88: [debug] plugin install hook for "whatup" exited with error
main.runHook
	helm.sh/helm/v3/cmd/helm/plugin.go:67
main.(*pluginInstallOptions).run
	helm.sh/helm/v3/cmd/helm/plugin_install.go:88
main.newPluginInstallCmd.func3
	helm.sh/helm/v3/cmd/helm/plugin_install.go:59
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.1.3/command.go:852
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.1.3/command.go:960
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.1.3/command.go:897
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:87
runtime.main
	runtime/proc.go:225
runtime.goexit
	runtime/asm_arm64.s:1130
```

**Comment by l0nax** [comment]
Ok, thanks, this is very useful.

The bug is fixed with d05fe35b44d92513544a73f3bca7c9bfe416382c.

**Comment by prikesh-patel** [comment]
That works for me now thank you! The plugin gets installed and I am able to run a `helm whatup`

Thanks for your help
